docs/contribute/locally.md

```diff
@@ -35,7 +35,7 @@ This guide uses the first option. If you'd like to clone the repository and buil
    Run this command to download and install the latest version of `docs-builder`:
 
    ```sh
-   sudo curl -L https://raw.githubusercontent.com/elastic/docs-builder/refs/heads/main/install.sh | sh
+   sudo curl -sL https://ela.st/docs-builder-install | sh
```

Comment:
Aside, this is the wrong way round, you want the `sudo` to be for executing it, not downloading it:

```suggestion
   curl -sL https://ela.st/docs-builder-install | sudo sh
```